test/Renci.SshNet.Tests/Classes/Common/HostKeyEventArgsTest.cs

```diff
@@ -45,6 +45,10 @@ public void HostKeyEventArgsConstructorTest()
             Assert.AreEqual(2048, target.KeyLength);
         }
 
+// Excluding on net462 platform, because using MD5 hash throws: System.InvalidOperationException:
+// 'This implementation is not part of the Windows Platform FIPS validated cryptographic algorithms.'
+#if !NET462_TARGET_FRAMEWORK
```

Comment:
I'll have to check this on my machine because it seems to be working now.

Reply:
Yes, I agree because this is not a part of this PR. @realvizu Can you revert this?

Reply:
Sure, reverted it.